lib/graphql/language/nodes.rb

```diff
@@ -202,7 +202,7 @@ class FragmentDefinition < AbstractNode
         def initialize_node(name: nil, type: nil, directives: [], selections: [])
           @name = name
           @type = type
-          @directives = directives
+          @directives = directives || []
```

Comment:
:+1: I'd say if the tests are green without this, then let's not add them. If "people" pass `nil`, they get nil :P (fortunately I think "people" is just `parser.rb`)